Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -757,6 +757,37 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .AddParameter("object", _("Object"), "Drawer")
       .AddParameter("expression", _("X drawing position"))
       .AddParameter("expression", _("Y drawing position"));
-
+  obj.AddAction("SetAntialiasing",
+                  _("Set antialiasing on the shape drawer"),
```

Comment:
This action could be replaced by a `None` choice in the "quality" property.

Reply:
I thought it was more clear to do it that way but I'll consider it !